kwc-modal.html

```diff
@@ -118,6 +121,7 @@
 </dom-module>
 
 <script>
+    console.warn('The `kwc-modal` element is deprecated. Visit https://github.com/KanoComponents/kwc-dialog to see how to migrate to kwc-dialog');
```

Comment:
Bit of a nit-pick here, but the other PR doesn't add a section into the README that would explain how to migrate.